actions/kv_grep.py

```diff
@@ -6,11 +6,14 @@
 
 
 class St2KVPGrepAction(St2BaseAction):
-    def run(self, query):
-        _keys = self.client.keys.get_all()
-        results = {}
-        for key in _keys:
-            if query in key.name:
-                results[key.name] = key.value
-
+    def run(self, query, prefix):
```

Comment:
Should add a default value for the ``prefix`` argument (``False``)..

Reply:
done